tex/gloss-korean.ldf

```diff
@@ -109,7 +135,7 @@ end
 local korean_break = function (head)
     local curr = head
     while curr do
-        if curr.id == 37 then
+        if curr.id == 37 and node.has_attribute(curr, attr_korean) then
```

Comment:
Using the numeric value for the node ID is a little fragile and not very easy to read; the recommended way to do so in LuaTeX (suggested by Taco) is `local glyph_id = node.id('glyph')` and then use `gyph_id` instead of 37 everywhere.